File: benchmark/corpus/10_control/a_expense_total.py
# Sums a list of expenses and prints the total. Nothing to flag.


def total_of(items):
    total = 0
    for label, amount in items:
        if amount > 0:
            total = total + amount
    return total


items = [("chai", 2500), ("book", 45000), ("auto", 12000)]
print("%d expense(s), total %d" % (len(items), total_of(items)))

File: benchmark/corpus/10_control/a_expense_total.js
// Sums a list of expenses and prints the total. Nothing to flag.
function totalOf(items) {
  let total = 0;
  for (const item of items) {
    if (item.amount > 0) total = total + item.amount;
  }
  return total;
}

const items = [
  { label: "chai", amount: 2500 },
  { label: "book", amount: 45000 },
  { label: "auto", amount: 12000 },
];
console.log(items.length + " expense(s), total " + totalOf(items));
